packages/dara-components/js/smart/chat/chat.tsx

```diff
@@ -96,14 +99,19 @@ function getAllUsersInChat(chat: Message[]): UiUserData[] {
 const anonymousUser: UserData = { identity_name: 'Anonymous' };
 
 /**
- * Api call to send the new message notification
+ * Api call to send the new message payload
  */
-async function sendNotification(payload: MessageNotificationPayload): Promise<void> {
+async function sendNewMessage(payload: MessageNotificationPayload, extras: RequestExtras): Promise<void> {
     try {
-        const res = await request('/notification/messages', {
-            body: JSON.stringify(payload),
-            method: HTTP_METHOD.POST,
-        });
+        const res = await request(
+            '/api/chat/messages',
+            {
+                body: JSON.stringify(payload),
+                method: HTTP_METHOD.POST,
+            },
+            extras
+        );
+        await handleAuthErrors(res);
```

Comment:
Should provide `true` as the second argument to redirect to login on an error rather than to the error page